news/2017-09-14-mapping-with-rewards-returns.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "Mapping With Rewards Returns"
+date: 2017-09-14 18:00:00 +0000
+---
+
+New and improved, the open-form **Mapping With Rewards** mapping contests are back! Map a selection of [Featured Artist](https://osu.ppy.sh/beatmaps/artists) tracks for your shot at a variety of prizes, from osu!store credit, osu!supporter and even custom profile badges and titles!
```

Comment:
serial comma, unless osu!supporter and the miscellaneous things _are_ a part of the osu!store credit 🤷‍♀️ 
`osu!supporter and even` ->
`osu!supporter, and even`